WordPress/src/main/java/org/wordpress/android/viewmodel/history/HistoryViewModel.kt

```diff
@@ -107,7 +117,10 @@ class HistoryViewModel @Inject constructor(
 
         revisionAuthorsId = ArrayList(revisionAuthorsId.distinct())
         _revisions.value = getHistoryListItemsFromRevisionModels(revisions)
-        fetchRevisionAuthorDetails(revisionAuthorsId)
+
+        if (revisionAuthorsId.isNotEmpty()) {
```

Comment:
I had to add this check in order to make this part of the class testable. We're currently only testing for empty `revisionAuthorsId`. If it's not empty, the `fetchRevisionAuthorDetails` will crash in unit tests.